documents/validator.go

```diff
@@ -300,6 +300,29 @@ func anchoredValidator(repo anchors.AnchorRepository) Validator {
 	})
 }
 
+// latestVersionValidator checks if the document is the latest version
+func latestVersionValidator(repo anchors.AnchorRepository) Validator {
+	return ValidatorFunc(func(_, model Model) error {
+		nextID, err := anchors.ToAnchorID(model.NextVersion())
+		if err != nil {
+			return errors.New("failed to get anchorID: %v", err)
+		}
+
+		var empty [anchors.DocumentRootLength]byte
+		var rootByte [anchors.DocumentRootLength]byte
+
+		root, _, err := repo.GetAnchorData(nextID)
```

Comment:
Are we sure that the GetAnchorData returns nil error if anchor not found?

Reply:
yes. testworld test already passed